docs/scos/dev/front-end-development/202307.0/oryx/set-up-fulfillment-app.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Set up Fulfillment App
+description: Learn how to set up Fulfillment App using a boilerplate project
+last_updated: Now 13, 2023
+template: howto-guide-template
+---
+
+{% info_block warningBox %}
+
+Fulfillment App is currently running under an Early Access Release. Early Access Releases are subject to specific legal terms, they are unsupported and do not provide production-ready SLAs. They can also be deprecated without a General Availability Release. Nevertheless, we welcome feedback from early adopters on these cutting-edge, exploratory features.
```

Comment:
I don't think this is an Early Access. Regardless, I'd stay out of it in our technical docs. 

What about one of these, we merge this into our existing setup docs? 
1. Download boilerplate for your application 
  (here we'd add a section to select the boilerplate of choice)
2. Install the application
3. Run the application

By wiring it into the existing Oryx docs, we emphasize (hopefully) that Oryx is the same for both composable storefront and FA.

Reply:
Let's try, but in this case we will not have a separate instalation guide for FA, which is a separate product. So from one perspetive it's great for Oryx, but can be confusing for FA oriented customers probably. In any case I'll try to apply your suggestion and let's see how it works.